### PR TITLE
Address clippy suggestions.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: bb6bebf371cdd2060202e69858954b3dca171172
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 387cc8749744854fdd30f1bd068ae56a00b7e5cb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: bb7056a3732c58610ce27be72fe1183b1989233b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 3393155a47cc61c33b5ff351716f90bbc1e39c93
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: c8fea8800f6e125e2d68bc3ff95ef8ad130391e7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 3bac597b025b587fb6b7f4692d67dcd4f8c87672
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/hyperium/hyper/blob/master/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

File: src/status.rs [hunk 1]
```diff
@@ -523,15 +521,6 @@ impl PartialEq for StatusCode {
...
 
-impl Eq for StatusCode {}
-
-impl Clone for StatusCode {
-    #[inline]
-    fn clone(&self) -> StatusCode {
-        *self
-    }
-}
-
```
Comment (seanmonstar) [inline src/status.rs]: These 3 manual impls were because the auto derives noticeably affected compile times. Is that no longer the case?

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: bb6bebf371cdd2060202e69858954b3dca171172
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 387cc8749744854fdd30f1bd068ae56a00b7e5cb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: bb7056a3732c58610ce27be72fe1183b1989233b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 3393155a47cc61c33b5ff351716f90bbc1e39c93
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: c8fea8800f6e125e2d68bc3ff95ef8ad130391e7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/hyperium/hyper/blob/master/CONTRIBUTING.md

---
This message was auto-generated by https://gitcop.com

File: src/status.rs (hunk 1 above)
Comment (frewsxcv) [inline src/status.rs]: No idea, don't have enough time to look into it so I ditched that change

**Comment by seanmonstar** [comment]
Thanks!

**Pull request merged**